packages/core/src/components/HelpDrawer/HelpDrawer.jsx

```diff
@@ -0,0 +1,74 @@
+import Button from '../Button/Button';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export class HelpDrawer extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.titleRef = null;
+  }
+
+  componentDidMount() {
+    if (this.titleRef) this.titleRef.focus();
+  }
+
+  render() {
+    const {
+      title,
+      children,
+      onCloseClick,
+      footerBody,
+      footerTitle
+    } = this.props;
+    /* eslint-disable jsx-a11y/no-noninteractive-tabindex, react/no-danger */
+    return (
+      <div className="ds-c-help-drawer">
+        <div className="ds-c-help-drawer__header">
+          {/* The nested div below might seem redundant, but we need a
+            * separation between our sticky header, and the flex container
+            * so things display as expected when the body content overflows
+            */}
+          <div className="ds-u-fill--gray-lightest ds-u-padding--2 ds-u-display--flex ds-u-align-items--start">
+            <h3
+              ref={el => (this.titleRef = el)}
+              tabIndex="0"
+              className="ds-u-text--lead ds-u-margin-y--0 ds-u-margin-right--2"
+            >
+              {title}
+            </h3>
+            <Button
+              className="ds-u-margin-left--auto"
```

Comment:
I'd like to have an `aria-label` attribute added here, with a default string of "Close help drawer". This will probably need English and Spanish translations, and could be a `HelpDrawer` default prop. Screen reader users don't get a lot of context when "Close, button" is announced on focus. The suggested text would read "Close help drawer, button" which is more descriptive and doesn't change the visual design.